lib/index.js

```diff
@@ -0,0 +1,21 @@
+const R = require('ramda');
+const validators = require('./validators');
+
+const toExport = {};
+
+Object.keys(validators).forEach((funcName) => {
+  const func = validators[funcName];
+  toExport[funcName] = validators[funcName];
+  const boolFuncName = funcName.replace('validate', 'isValid');
+  toExport[boolFuncName] = async (...args) => {
+    let err;
+    try {
+      await func(...args);
+    } catch (e) {
+      err = e;
+    }
+    return !err;
+  };
```

Comment:
since you don't actually care about the error, you could write it like:

```javascript
toExport[boolFuncName] = async (...args) => {
  try {
    await func(...args);
    return true;
  } catch (e) {
    return false;
  }
};
```

Reply:
`async/await` is great and all, but I think this is a good time to just use Promises directly.

```js
toExport[boolFuncName] = compose(
  (x) => x.then(T, F),
  func
);
```

I'm assuming that `func` is a promise returning function. If it's not, you could just `Promise.resolve` the return to normalize it. This also assumes that your `func` isn't going to throw synchronously, and if it's a
promise returning function, it shouldn't.